web/src/components/PasswordWithMeter.js

```diff
@@ -27,7 +27,7 @@ class Password extends Component {
   };
 
   render() {
-    const { title, value, showMeter, ...rest } = this.props;
+    const { compareTo, title, value, showMeter, ...rest } = this.props;
```

Comment:
Need to pull out `compareTo`, otherwise it gets caught up in `...rest` and passed into a `<div>` which makes React very unhappy.

![](https://media.giphy.com/media/3t7RAFhu75Wwg/giphy.gif)